index.js

```diff
@@ -47,6 +47,15 @@ const worker = queue.async.asyncify(doWork => doWork());
 
 module.exports = Filter;
 
+module.exports.shouldPersist = shouldPersist;
+
+function shouldPersist(env, persist) {
+  let isCI = env.CI;
+  let forceCICaching = isCI && env.FORCE_PERSISTANCE_IN_CI;
+
+  return forceCICaching || persist && !isCI;
```

Comment:
Can you either add paren here to clarify order of operations or refactor to return separately? 

Maybe something like:

```js
function shouldPersist(env, persist) {
  let isCI = env.CI;
  let forceCICaching = isCI && env.FORCE_PERSISTANCE_IN_CI;

  if (forceCICaching) { return true; }

  return persist && !isCI;
}
```

Reply:
function shouldPersist(env, persist) {
    if (env.CI) {
        return env.FORCE_PERSISTENCE_IN_CI;
    } else {
        return persist;
    }
}

(if the above is correct, then 'return (env.CI ? env.FORCE_PERSISTANCE_IN_CI : persist);' is better.

Reply:
lol, i actually wrote the above speculatively trying to guess what you two would prefer. Guess i was off :P

Reply:
I think the one-liner is best. ;-)